index.html

```diff
@@ -0,0 +1,172 @@
+  <section class="program flex" id="program">
+    <h2>Main Program</h2>
+    <div class="program-items flex">
+      <div class="p-cards margin flex">
+        <div class="dp-img">
+          <img src="asset/defending.png" alt="defending"  class="p-icon">
+        </div>
+        <div class="dp-text margin">
+          <span class="p-text red">Defending</span>
+        </div>
+        <div class="dp-summary">
+          <span class="p-summary">Learn how to stay low, keep at arms length, good footwork and active hands </span>
+        </div>
+      </div>
+      <div class="p-cards margin flex">
+        <div class="dp-img">
+          <img src="asset/shooting.png" alt="shooting"  class="p-icon">
+        </div>
+        <div class="dp-text margin">
+          <span class="p-text red">Shooting</span>
+        </div>
+        <div class="dp-summary">
+          <span class="p-summary">Understand the fundermentals of shooting,learn freethrows, jumpshots, lay-ups and dunks.</span>
+        </div>
+      </div>
+      <div class="p-cards margin flex">
+        <div class="dp-img">
+          <img src="asset/dribbling.png" alt="dribbling"  class="p-icon">
+        </div>
+        <div class="dp-text margin">
+          <span class="p-text red">Dribbling</span>
+        </div>
+        <div class="dp-summary">
+          <span class="p-summary">Empower your dribbling skills,with our program as we take you through the basics of bouncing, change of pace, crossovers, between the legs and behind the back.</span>
+        </div>
+      </div>
+      <div class="p-cards margin flex">
+        <div class="dp-img">
+          <img src="asset/rebounding.png" alt="rebounding"  class="p-icon">
+        </div>
+        <div class="dp-text margin">
+          <span class="p-text red">Rebounding</span>
+        </div>
+        <div class="dp-summary">
+          <span class="p-summary">Learn basics of rebounding and increase your basketball IQ, knowing your stance boxing out and anticipating where the ball will be, but you have to put in the work.</span>
+        </div>
+      </div>
+      <div class="p-cards margin flex">
+        <div class="dp-img">
+          <img src="asset/passing.png" alt="passing"  class="p-icon">
+        </div>
+        <div class="dp-text margin">
+          <span class="p-text red">Passing</span>
+        </div>
+        <div class="dp-summary">
+          <span class="p-summary">Understand basics of the bounce pass, chest pass, overhead pass and baseball pass and know when to apply these in game. </span>
+        </div>
+      </div>
+    </div>
+    <button class="program-btn">Join ABB Camp Summit 2023</button>
+    <p class="program-text">SEE THE WHOLE PROGRAM</p>
+  </section>
+  <!-- Coaches -->
+  <section class="coaches flex" id="campaign">
+    <h2>Featured Coaches</h2>
+    <div class="coach-cards flex"></div>
+    <button type="reset" class="more-btn flex">
+      MORE<i class='bx bx-chevron-down'></i>
+    </button>
```

Comment:
- [ ] [REQUIRED] Please fix your `More` button because it is not working for me!

![image](https://user-images.githubusercontent.com/63915024/190244530-8916db99-9294-4a46-86d0-ab83b17e1dc5.png)

Reply:
this is not part of the requirements

Reply:
#### Hi @UsirusenIkon!
#### Sorry for forgetting that this was an optional requirement,
####  I changed it to `[OPTIONAL]`!